commit 6a5184d7f8f1a22b589a486835c24f0f435c4914 (refs/changes/40/2040/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2017-06-05T11:51:56-07:00 (2 years, 4 months ago)
    
    joyent/node-smartdc-auth#12 KeyRingHomeDirPlugin.listKeys() invokes callback twice on failure
    joyent/node-smartdc-auth#10 bump the signaturecache event emitter limit

diff --git a/package.json b/package.json
index bff9b76..be5945f 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "triton",
   "description": "Joyent Triton CLI and client (https://www.joyent.com/triton)",
-  "version": "5.2.0",
+  "version": "5.2.1",
   "author": "Joyent (joyent.com)",
   "homepage": "https://github.com/joyent/node-triton",
   "dependencies": {
@@ -20,7 +20,7 @@
     "restify-errors": "3.0.0",
     "rimraf": "2.4.4",
     "semver": "5.1.0",
-    "smartdc-auth": "2.5.2",
+    "smartdc-auth": "2.5.5",
     "sshpk": "1.10.2",
     "sshpk-agent": "1.4.2",
     "strsplit": "1.0.0",
